src/metrics.ts

```diff
@@ -11,6 +11,46 @@ export const metricDeclarations = {
     help: 'Count calls to ping',
     type: IMetricsComponent.CounterType,
     labelNames: ['pathname']
+  },
+  sns_publish_success: {
+    help: 'Count successful SNS publishes',
+    type: IMetricsComponent.CounterType,
+    labelNames: ['type']
```

Comment:
Soon we will remove this label because we are going to get rid of one of the two SNS topics, not sure if the label change will cause any impact on the scrapping